### PR TITLE
e2e tests evaluation in CI

### PR DESCRIPTION
**Pull request by VineethReddy02**
1. Added e2e tests to CI
2. Changed the K8s tool from minikube to kind for e2e tests. As kind (k8s on Docker) is more light weight and easy run in local development machines. Added `e2e-tests.sh` script this creates kind cluster, runs e2e tests, & deletes kind cluster. Dropped minikube start, stop & delete logic from e2e tests.
3. Point the tobs in e2e tests to tobs in the bin directory as the latest tobs exe is placed in the bin directory after the build.
4. Comment memory requests in testdata files as `2Gi` requests lag underlying minikube/kind clusters while resource creation.
5. Comment tobs concurrent tests as this needs some re-work in running these tests.
6. Add badges to tobs README.md

Signed-off-by: Vineeth Pothulapati <vineethpothulapati@outlook.com>

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -119,8 +119,6 @@ Then, move the `tobs` binary from the current directory to your `/bin` folder.
 
 ## Testing
 
-WARNING: Tests start, stop, and delete the active minikube cluster. Make sure it's safe to delete your minikube cluster before starting the test.
+__Dependencies__: [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), [kind](https://kind.sigs.k8s.io/)
 
-A testing suite is included in the `tests` folder. This testing suite has additional dependencies on [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
-
-The testing suite can be run by calling `go test -timeout 30m` from within the `tests` folder. At least 4 cpus should be allocated for minikube with e.g. `minikube config set cpus 4`.
+A testing suite is included in the `tests` folder. The testing suite can be run by `./e2e-tests.sh` this script will create kind cluster executes the test suite and deletes the kind cluster.
```
Comment (blagojts) [inline cli/README.md:124]:
```suggestion
A testing suite is included in the `tests` folder. The testing suite can be run by `./e2e-tests.sh` this script will create a [kind](https://kind.sigs.k8s.io) cluster, execute the test suite, and delete the kind cluster.
```

File: cli/tests/concurrent_test.go [hunk 2]
```diff
@@ -30,6 +30,13 @@ func TestConcurrent(t *testing.T) {
 		t.Skip("Skipping concurrent tests")
 	}
 
+	// skipping concurrent tests for now
```
Comment (blagojts) [inline cli/tests/concurrent_test.go:33]: Maybe it's better to just do a t.Skip("reason why") so we get it logged when running the tests

File: cli/tests/testdata/f1.yml [hunk 3]
```diff
@@ -46,8 +46,8 @@ promscale:
     requests:
       # By default this should be enough for a cluster
       # with only a few pods
-      memory: 2Gi
-      cpu: 1
+      #memory: 2Gi
+      #cpu: 1
```
Comment (blagojts) [inline cli/tests/testdata/f1.yml:50]: remove rather than have it commented

**Review by blagojts — APPROVED** [review]
Great changes. I left a few small suggestions

**Review by blagojts — APPROVED** [review]

File: cli/e2e-tests.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+kind create cluster
+go test -v ./tests/ --timeout 30m
+kind delete cluster
```
Comment (cevian) [inline cli/e2e-tests.sh:5]: Maybe we should put this in a trap so it happens if the test is cancelled  or interrupted with ctrl-c or whatnot?

**Review by cevian — APPROVED** [review]
Looks great. 1 small (optional) suggestion

**Pull request merged**